en/flight_modes_rover/manual.md

```diff
@@ -0,0 +1,88 @@
+# Manual Modes (Rover)
+
+Manual modes require stick inputs from the user to drive the vehicle.
+
+![Manual Controls](../../assets/airframes/rover/flight_modes/rover_manual_controls.png)
+
+The manual modes listed below provide increasing levels of autopilot support:
+
+| Mode                           | Features                                                                                                   |
+| ------------------------------ | ---------------------------------------------------------------------------------------------------------- |
+| [Manual](#manual-mode)         | Directly map stick inputs to motor commands, no closed loop control.                                       |
+| [Acro](#acro-mode)             | Closed loop yaw rate control.                                                                              |
+| [Stabilized](#stabilized-mode) | Closed loop yaw rate and yaw control.                                                                      |
+| [Position](#position-mode)     | Closed loop yaw rate, yaw, and speed control, and uses GPS coordinates for improved straight line driving. |
+
+## Manual Mode
+
+<Badge type="tip" text="Differential" /><Badge type="tip" text="Ackermann" />
+
+The _Manual_ mode stops the rover when the RC control sticks are centred.
+To manually move/drive the vehicle you move the sticks outside of the centre.
+
+- [**Differential Rover**](../frames_rover/differential_rover.md): Moving the left-stick up/down controls the _forward speed_ and moving the right-stick left/right controls the _yaw rate_ of the vehicle.
+
+- [**Ackermann Rover**](../frames_rover/ackermann_rover.md): Moving the left-stick up/down controls the _forward speed_ and moving the right-stick left/right controls the _steering angle_ of the vehicle.
```

Comment:
This is true for all modes right? The difference is that in some modes you do closed loop control over the rate/angle?

Reply:
Well the end-effect is the same, but in some modes the stick inputs are mapped to motor commands and in some they are mapped to setpoints for the control system. I tried to clarify this with the tables for the stick inputs i added [here](https://github.com/PX4/PX4-user_guide/pull/3377/files#diff-dd65fbcd095c40befe925ee616cdb0679de431aac441983568baf7b8e23ed9c5R28-R94).